scripts/gulpfile.factory.js

```diff
@@ -76,11 +77,16 @@ module.exports = function factory({
         cwd: paths.source
       })
         .pipe(sourcemaps.init())
-        .pipe(sass().on("error", sass.logError))
+        .pipe(
+          sass({
```

Comment:
@starryeyez024 Include paths are built into node-sass so you can use:

```
sass({
  includePaths: ["node_modules"]
})
```

Instead of bringing in a new dependency to do this.